packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx

```diff
@@ -6,7 +6,7 @@ import { Dropdown, IDropdownOption } from 'office-ui-fabric-react/lib/Dropdown';
 import { List, ScrollToMode } from 'office-ui-fabric-react/lib/List';
 import { TextField } from 'office-ui-fabric-react/lib/TextField';
 import { Checkbox } from 'office-ui-fabric-react/lib/Checkbox';
-import * as styles from './List.Scrolling.Example.scss';
+import { ITheme, getTheme, mergeStyleSets, FontSizes, normalize } from 'office-ui-fabric-react/lib/';
```

Comment:
Should be like this:
```suggestion
import { ITheme, getTheme, mergeStyleSets, FontSizes, normalize } from 'office-ui-fabric-react/lib/Styling';
```